relnotes/array.feature.md

```diff
@@ -0,0 +1,26 @@
+### Non-allocating replacement for declaring array literals.
+
+* `ocean.core.Array`
+
+  Add a new helper function `fill()` to assign all the elements of an array.
+  The function is a replacement for declaring an array literal, it does not
```

Comment:
Suggested rewording to clarify the rationale for this:
"
The function is an alternative for declaring an array literal, which is known to allocate GC memory. (`fill()` also checks that the length of the array matches the number of variadic arguments.)
"

Reply:
Sounds good, I'll update it